source/wp-content/themes/wporg-developer-2023/src/article-meta-github/block.json

```
@@ -8,7 +8,20 @@
 	"icon": "smiley",
 	"description": "",
 	"usesContext": [ "postId", "postType" ],
-	"attributes": {},
+	"attributes": {
+		"linkUrl": {
+			"type": "string",
+			"default": ""
+		},
+		"heading": {
+			"type": "string",
+			"default": ""
+		},
+		"text": {
```

Maybe this would make the purpose of the content clearer
```suggestion
		"linkText": {
```